### PR TITLE
Add desdeo-vis

### PR DESCRIPTION
**Pull request by frankier**

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please relint.

**Comment by jakirkham** [comment]
Thanks @frankier :)

File: recipes/desdeo-vis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
+    - python >= 3.6
```
Comment (jakirkham) [inline recipes/desdeo-vis/meta.yaml]: So should have caught this, but I missed it. There shouldn't be a space after `>=`.

File: recipes/desdeo-vis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - ipywidgets >=7.0.0
+    - widgetsnbextension
+  run:
+    - python >= 3.6
```
Comment (jakirkham) [inline recipes/desdeo-vis/meta.yaml]: Same here.

File: recipes/desdeo-vis/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipes/desdeo-vis/meta.yaml]: Fixed in PR ( https://github.com/conda-forge/staged-recipes/pull/6311 ).

File: recipes/desdeo-vis/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  imports:
+    - desdeo_vis
+    - desdeo_vis.nbextension
+    - desdeo_vis.plot
+    - desdeo_vis.widget
+
+about:
+  home: https://github.com/industrial-optimization-group/desdeo-vis
+  license: MPL-2.0
+  license_file: 'LICENSE'
```
Comment (jakirkham) [inline recipes/desdeo-vis/meta.yaml]: Despite it passing here, apparently the license file cannot be found in the feedstock build.

ref: https://circleci.com/gh/conda-forge/desdeo-vis-feedstock/2

**Comment by jakirkham** [comment]
Hmm...so apparently if the recipe is deemed invalid, the `staged-recipes` build skips it and passes. Seems like it should fail instead. Thoughts @conda-forge/core @conda-forge/staged-recipes?

```
invalid recipe dir: /home/conda/conda-recipes/desdeo-vis - skipping
Computed that there are 0 distributions to build from 1 recipes
Nothing to do
```

ref: https://travis-ci.org/conda-forge/staged-recipes/builds/405406151#L313-L315

**Comment by frankier** [comment]
Thanks for the notes + fixes @jakirkham . I've fixed the other problem now. Good luck fixing the CI.

**Pull request merged**